postgres/postgresql.conf.erb

```diff
@@ -33,13 +32,20 @@ lc_numeric = 'en_AU.UTF-8'
 lc_time = 'en_AU.UTF-8'
 default_text_search_config = 'pg_catalog.english'
 
+idle_in_transaction_session_timeout = '120s'
+
 # These settings only apply to the master.
-wal_level = hot_standby
+wal_level = replica
```

Comment:
@josh we have the option of changing this from `replica` to `logical`. That will make the WAL files larger and increase data transfer, but also gives us the future option of experimenting with logical replication. In particular, we'd have the option of trying [pglogical](https://2ndquadrant.com/en/resources/pglogical/) for the upgrade to psql 10 with reduced downtime.

What do you think? Is it worth the extra data transfer "just in case"?